attest/verifier/types/src/convert/quote3.rs

```diff
@@ -6,11 +6,11 @@ use crate::{prost, ConversionError};
 use alloc::vec::Vec;
 use mc_sgx_dcap_types::Quote3;
 
-impl TryFrom<prost::Quote3> for Quote3<Vec<u8>> {
+impl TryFrom<&prost::Quote3> for Quote3<Vec<u8>> {
     type Error = ConversionError;
 
-    fn try_from(value: prost::Quote3) -> Result<Self, Self::Error> {
-        Ok(Quote3::try_from(value.data)?)
+    fn try_from(value: &prost::Quote3) -> Result<Self, Self::Error> {
+        Ok(Quote3::try_from(value.data.clone())?)
```

Comment:
tradeoff: A clone that didn't exist, but it should be cheaper than the caller cloning all of `DcapEvidence`